### ISSUE TITLE
New locale issue in pelican-quickstart in 3.7.1

### ISSUE THREAD

**Issue by ogenstad** [getpelican/pelican#2104]
On my system I don't have the LANG variable set but do have the LC_CTYPE.

```
patrick at ns2803m in ~/src/pelican (pelican)
$ pelican --version
3.7.1

patrick at ns2803m in ~/src/pelican (pelican)
$ set | grep -E "LANG|LC_TYPE"

patrick at ns2803m in ~/src/pelican (pelican)
$ set | grep -E "LANG|LC_CTYPE"
LC_CTYPE=UTF-8

patrick at ns2803m in ~/src/pelican (pelican)
$ pelican-quickstart
Traceback (most recent call last):
  File "/Users/patrick/.virtualenvs/pelican/bin/pelican-quickstart", line 7, in <module>
    from pelican.tools.pelican_quickstart import main
  File "/Users/patrick/.virtualenvs/pelican/lib/python2.7/site-packages/pelican/tools/pelican_quickstart.py", line 25, in <module>
    _DEFAULT_LANGUAGE = locale.getlocale()[0]
  File "/Users/patrick/.virtualenvs/pelican/lib/python2.7/locale.py", line 564, in getlocale
    return _parse_localename(localename)
  File "/Users/patrick/.virtualenvs/pelican/lib/python2.7/locale.py", line 477, in _parse_localename
    raise ValueError, 'unknown locale: %s' % localename
ValueError: unknown locale: UTF-8

patrick at ns2803m in ~/src/pelican (pelican)
$ unset LC_CTYPE

patrick at ns2803m in ~/src/pelican (pelican)
$ pelican-quickstart
Welcome to pelican-quickstart v3.7.1.

This script will help you create a new Pelican-based website.

Please answer the following questions so this script can generate the files
needed by Pelican.


> Where do you want to create your new web site? [.]
```

**Comment by avaris** [comment]
Probably related to this [python bug](https://bugs.python.org/issue18378)

**Comment by coolum001** [comment]
I fixed this by adding the line

locale.setlocale(locale.LC_ALL,"")

before the conf dict setup in pelican-quickstart.py

so that the line
locale.getlocale()[0].split('_')[0]

now works OK

in my case returning

'English'

I think (reading the python locale docn) that adding the locale.LC_ALL to the getlocale call  might also work, but I haven't tested this.

**Comment by avaris** [comment]
@coolum001 that line is already [there](https://github.com/getpelican/pelican/blob/3.7.1/pelican/tools/pelican_quickstart.py#L24) in 3.7.1. Are you sure you have 3.7.1 and not 3.7.0?

**Comment by coolum001** [comment]
Sorry about this - yes I have pelican 3.7.0;  downloaded from conda a few days ago, and I assumed that I had the latest.  Anaconda Navigator says pelican 3.7.0 is 'not upgradeable', by the way?

Sorry for the distraction (but for folks like me on conda, it might be useful info?)

Cheers

**Comment by justinmayer** [comment]
Issue closed, as it is already addressed in 3.7.1.
